docs/sphinx/source/conf.py

```diff
@@ -86,6 +87,14 @@
 
 master_doc = 'index'
 
+# Example configuration for intersphinx: refer to the Python standard library.
+intersphinx_mapping = {
+    'python': ('https://docs.python.org/3.8/', None),
+    'pandas': ('https://pandas.pydata.org/pandas-docs/stable/', None),
+    'numpy': ('https://numpy.org/doc/stable/', None),
+    'scipy': ('https://docs.scipy.org/doc/scipy/reference/', None'),
```

Comment:
```suggestion
    'scipy': ('https://docs.scipy.org/doc/scipy/reference/', None),
```
This should fix some of the failing tests